WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -1419,6 +1432,7 @@ class MySiteViewModel @Inject constructor(
                     mapOf(MY_SITE_TAB to MySiteTabType.DASHBOARD.trackingLabel)
             )
             analyticsTrackerWrapper.track(Stat.MY_SITE_DASHBOARD_SHOWN)
+            trackBloggingPromptCardShown()
```

Comment:
While testing this I noticed this is only called when the tab is actually changed, so if the app already opens at the expected tab (My Site -> Home) this is not called.

Reply:
Thanks for catching that! As we've discussed in DM, I wasn't able to reproduce this one yet 🤔

> maybe trackTabChanged is only called when actually changing tabs or something. 🤔

I went through the code and this is what I've found:
```
        override fun onPageSelected(position: Int) {
            super.onPageSelected(position)
            viewModel.onTabChanged(position)
        }
```
This callback is being triggered even the first time the screen is loaded, so in my tests the tracking was also being called the first time the screen was loaded.

I've also tried to look for any race condition, but it seems like we're getting the selected site (which is needed to verify if the prompt card is loaded) synchronously. It's value is being stored in memory by a singleton called `SelectedSiteRepository` 🤔

Reply:
We discussed that further in a DM and I will tackle this in a different PR.